timeseries/src/autogluon/timeseries/models/local/statsmodels.py

```diff
@@ -54,74 +82,71 @@ class ETSModel(AbstractStatsmodelsModel):
         When set to -1, all CPU cores are used.
     """
 
-    quantile_method_name = "pred_int"
-    statsmodels_allowed_init_args = [
+    allowed_local_model_args = [
         "error",
         "trend",
         "damped_trend",
         "seasonal",
         "seasonal_period",
-    ]
-    statsmodels_allowed_fit_args = [
         "maxiter",
     ]
 
-    def _update_sm_model_init_args(self, sm_model_init_args: dict, data: TimeSeriesDataFrame) -> dict:
-        sm_model_init_args = sm_model_init_args.copy()
-        sm_model_init_args["freq"] = data.freq
-        sm_model_init_args.setdefault("trend", "add")
-
-        # Infer seasonal_period if seasonal_period is not given / is set to None
-        seasonal_period = sm_model_init_args.pop("seasonal_period", None)
-        if seasonal_period is None:
-            seasonal_period = get_seasonality(data.freq)
-        sm_model_init_args["seasonal_periods"] = seasonal_period
+    def _update_local_model_args(
+        self, local_model_args: Dict[str, Any], data: TimeSeriesDataFrame, **kwargs
+    ) -> Dict[str, Any]:
+        local_model_args.setdefault("trend", "add")
+        local_model_args.setdefault("maxiter", 1000)
 
-        seasonal = sm_model_init_args.setdefault("seasonal", "add")
-        # Disable seasonality if seasonal_period is too short
+        seasonal_period = local_model_args.pop("seasonal_period")
+        seasonal = local_model_args.setdefault("seasonal", "add")
         if seasonal is not None and seasonal_period <= 1:
             logger.warning(
                 f"{self.name} with seasonal = {seasonal} requires seasonal_period > 1 "
                 f"(received seasonal_period = {seasonal_period}). Disabling seasonality."
             )
-            sm_model_init_args["seasonal"] = None
-            sm_model_init_args["seasonal_periods"] = 1
+            local_model_args["seasonal"] = None
+            local_model_args["seasonal_periods"] = 1
+        else:
+            local_model_args["seasonal_periods"] = seasonal_period
 
-        return sm_model_init_args
+        return local_model_args
+
+    @staticmethod
+    def _predict_with_local_model(
+        time_series: pd.Series,
+        freq: str,
+        prediction_length: int,
+        quantile_levels: List[float],
+        local_model_args: dict,
+        **kwargs,
+    ) -> pd.DataFrame:
+        forecast_timestamps = get_forecast_horizon_index_single_time_series(
+            past_timestamps=time_series.index, freq=freq, prediction_length=prediction_length
+        )
+        from statsmodels.tsa.exponential_smoothing.ets import ETSModel
```

Comment:
any particular reason we move the imports in?

Reply:
Will make our life easier should we move to soft dependency on statsmodels? But can also be placed outside. What do you think?